src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -47,16 +47,24 @@ Complete some one-off setup tasks on your computer and Android device.
 
 On your development computer:
 
-- Install or update [web-ext](https://github.com/mozilla/web-ext) to version 4.1.0 or later.
+- Install or update [web-ext](https://github.com/mozilla/web-ext) to version 7.12.0 or later.
```

Comment:
For future reference, the 7.12 recommendation here follows from the fix needed to support current Android versions, implemented in https://github.com/mozilla/web-ext/issues/2312